localized/en-us/games/eggquiz/eggquizmath_levels.tsv

```diff
@@ -389,37 +388,37 @@ SpecialTest_21	1	210001, 210002, 210003, 210003, 210004
 			230007	word_problem	×æçÚØæ Ùð 3 ç·UUUUÌæÕð¢ ÂÉ¸è¢ ¥æñÚ SÅðâè Ùð 5 ç·UUUUÌæÕð¢ ÂÉ¸è¢Ð ×æçÚØæ ·UUUè ÌéÜÙæ ×ð¢ SÅðâè Ùð ç·UUUUÌÙè ¥æñÚ ç·UUUUÌæÕð¢ ÂÉ¸è¢?	मारिया ने ३ किताबें पढ़ीं और स्टेसी ने ५ किताबें पढ़ीं। मारिया की तुलना में स्टेसी ने कितनी और किताबें पढ़ीं?				2				
 			230008	word_problem	çâÇÙè ·UUUUô â×é¼ý ÌÅ ÂÚ 20 àæ¢¹ ç×ÜðÐ Ùô¥æ ·UUUUô 6 àæ¢¹ ç×ÜðÐ Ùô¥æ ·UUUè ÌéÜÙæ ×ð¢ çâÇÙè ·UUUUô ç·UUUUÌÙð ¥æñÚ àæ¢¹ ç×Üð?	सिडनी को समुद्र तट पर २० शंख मिले। नोआ को ६ शंख मिले। नोआ की तुलना में सिडनी को कितने और शंख मिले?				14				
 			230009	word_problem	¥óææ Ùð 12 ÂõÏð ¹ÚèÎðÐ ©â·UUUè ÕãÙ Ùð 7 ÂõÏð ¹ÚèÎðÐ ¥óææ Ùð ¥ÂÙè ÕãÙ âð ç·UUUUÌÙð ¥çÏ·UUUU ÂõÏð ¹ÚèÎð?	अन्ना ने १२ पौधे खरीदे। उसकी बहन ने ७ पौधे खरीदे। अन्ना ने अपनी बहन से कितने अधिक पौधे खरीदे?				5				
 			230010	word_problem	Åæò× Ùð S·êUUUÜ ·ðUUU Õ»è¿ð ×ð¢ 5 ÕèÁ Ü»æ°Ð ÁðÙ Ùð 8 ÕèÁ Ü»æ°Ð Åæò× Ùð ÁðÙ ·UUUè ÌéÜÙæ ×ð¢ ç·UUUUÌÙð ·UUUU× ÕèÁ Ü»æ°?	टॉम ने स्कूल के बगीचे में ५ बीज लगाए। जेन ने ८ बीज लगाए। टॉम ने जेन की तुलना में कितने कम बीज लगाए?				3				
 SpecialTest_24	1	240001~240010, 240001~240010, 240001~240010, 240001~240010, 240001~240010												
 			240001	word_problem	×ðÚð Âæâ 10 âðÕ ãñ¢Ð ×ñ¢ ·éUUUÀ âðÕ ¹æÌæ ãê¢Ð ×ðÚð Âæâ 4 âðÕ Õ¿ð ãñ¢Ð ×ñ¢Ùð ç·UUUUÌÙð âðÕ ¹æ°?	मेरे पास १० सेब हैं। मैं कुछ सेब खाता हूं। मेरे पास ४ सेब बचे हैं। मैंने कितने सेब खाए?				6				
-			240002	word_problem	·UUUUÜ, Îé·UUUæÙ ×ð¢ ¿æßÜ ·ðUUU 7 Õñ» ÍðÐ ¥æÁ, Îé·UUUæÙ ×ð¢ ¿æßÜ ·ðUUU 2 Õñ» ãñ¢Ð Îé·UUUæÙ ×ð¢ ç·UUUUÌÙð Õñ» ¿æßÜ çÕ·UUUU »°?	कल, दुकान में चावल के ७ बैग थे। आज, दुकान में चावल के २ बैग हैं। दुकान में कितने बैग चावल बिक गए?				5				
-			240003	word_problem	ÇðçßÇ ¥æñÚ ©â·ðUUU ÎôSÌ 9 ÂÌ¢» ©Ç¸æÌð ãñ¢Ð ·éUUUÀ ÂÌ¢» °·UUUU ÂðÇ¸ ×ð¢ È¢UUUâ ÁæÌè ãñ¢Ð 6 ÂÌ¢»ð¢ ¥Öè Öè ©Ç¸ Úãè ãñ¢Ð ÂðÇ¸ ×ð¢ ç·UUUUÌÙè ÂÌ¢»ð¢ ¥Å·UUUè ãñ¢?	डेविड और उसके दोस्त ९ पतंग उड़ाते हैं। कुछ पतंग एक पेड़ में फंस जाती हैं। ६ पतंगें अभी भी उड़ रही हैं। पेड़ में कितनी पतंगें अटकी हैं?				3				
-			240004	word_problem	°·UUUU ¥æÎ×è »éyÕæÚð Üð Áæ Úãæ Íæ Üðç·UUUUÙ ãßæ Ùð 6 ©Ç¸æ çÎ°Ð ©â·ðUUU Âæâ 2 »éyÕæÚð Õ¿ð ãñ¢Ð ©â·ðUUU Âæâ ÂãÜð  ç·UUUUÌÙð »éyÕæÚô¢ Íð?	एक आदमी गु4बारे ले जा रहा था लेकिन हवा ने ६ उड़ा दिए। उसके पास २ गु4बारे बचे हैं। उसके पास पहले  कितने गु4बारों थे?				8				
+			240002	word_problem	·UUUUÜ, Îé·UUUæÙ ×ð¢ ¿æßÜ ·ðUUU 7 Õñ» ÍðÐ ¥æÁ, Îé·UUUæÙ ×ð¢ ¿æßÜ ·ðUUU 2 Õñ» ãñ¢Ð Îé·UUUæÙ ×ð¢ ç·UUUUÌÙð Õñ» ¿æßÜ çÕ·UUUU »°?	कल दुकान में चावल के ७ बैग थे। आज दुकान में चावल के २ बैग हैं। दुकान में कितने बैग चावल बिक गए?				5				
+			240003	word_problem	ÇðçßÇ ¥æñÚ ©â·ðUUU ÎôSÌ 9 ÂÌ¢» ©Ç¸æÌð ãñ¢Ð ·éUUUÀ ÂÌ¢» °·UUUU ÂðÇ¸ ×ð¢ È¢UUUâ ÁæÌè ãñ¢Ð 6 ÂÌ¢»ð¢ ¥Öè Öè ©Ç¸ Úãè ãñ¢Ð ÂðÇ¸ ×ð¢ ç·UUUUÌÙè ÂÌ¢»ð¢ ¥Å·UUUè ãñ¢?	डेविड और उसके दोस्त ९ पतंग उड़ाते हैं। कुछ पतंग एक पेड़ में फस जाती हैं। ६ पतंगें अभी भी उड़ रही हैं। पेड़ में कितनी पतंगें अटकी हैं?				3				
```

Comment:
Update Chanakya text

Reply:
What needed to be changed in Chanakya?

Reply:
The फंस text in the Chanakya encoding is not updated.